File: 2_variables/variables.c
#include <stdio.h>
#include <stdbool.h> // libreria para el dato bool
#define d 10 //definiendo variable constante, tambien se puede con la palabra const

// definiendo variable de tipado estatico
int a = 1;

// definiendo multiples variables
int b = 10, c = 0;

bool e = true;

float f = 1.55;

char g[10] = "hola mundo";

int main(){

    a = a*10;

    printf("%i \n", a+b+c+d);
    printf("%i \n", d);
    printf("%d \n", e);
    printf("%f \n", f);
    printf("%s \n", g);
    

    return 0;
}
      

File: 02_variables/variables.go
package main

import "fmt"


// declarando variable de tipado estatico
var a int

// declarando variable estatica

const c int = 10

// declarando multiples variables

var (
    d bool
    e float32
    f string
)

func main(){

    a = 10

    //declarando variable de tipado estatico
    b := 10

    d, e = true, 1.55
    f = "hola mundo"

    fmt.Println(a+b+c)
    fmt.Println(c)
    fmt.Println(d)
    fmt.Println(e)
    fmt.Println(f)

}





